apps/ff-site/src/app/_styles/globals.css

```diff
@@ -25,6 +25,10 @@
   --color-icon-error: theme('colors.brand.error');
 
   --spacing-readable: 60ch;
+
+  --tooltip-background-color: var(--color-brand-200);
+  --tooltip-color: var(--color-brand-800);
+  --tooltip-arrow-color: var(--color-brand-200);
```

Comment:
Just checking - did we not decide to use classnames?

Reply:
Correct 👍 I pushed a fix